src/rcheevos/consoleinfo.c

```diff
@@ -675,15 +690,21 @@ const rc_memory_regions_t* rc_console_memory_regions(int console_id)
     case RC_CONSOLE_ORIC:
       return &rc_memory_regions_oric;
 
+    case RC_CONSOLE_PC6000:
+      return &rc_memory_regions_pc6000;
```

Comment:
You'll need to leave these cases out of the switch statement until the related data is available. They're causing compilation errors (see failing checks)